scripts/build_node_package.sh

```diff
@@ -26,6 +20,10 @@ export NAN_PATH
 sed -i 's/node \-e \\"require('\''nan'\'')\\"/echo $NAN_PATH/g' binding.gyp
 rm -rf nan.tar.gz
 mkdir $BUILD_TARGET_DIR
+if [ "$(uname -m)" = "aarch64" ] && [ "$(< /etc/os-release grep VERSION | head -n1 | cut -d \" -f2)" = "8" ]; then
```

Comment:
```suggestion
if [ "$(uname -m)" = "aarch64" ] && [ "$(grep VERSION /etc/os-release | head -n1 | cut -d \" -f2)" = "8" ]; then
```
More readable :) Also, whenever you're writing parsing code, it's nicer if you explain in comment what is the expected input you're going to parse, and what is the expected result, and why are you parsing. So - an example line of "VERSION", and the reason that we're trying to extract the CentOS version, to check if 8.

Reply:
Btw in one site we use `grep -q "CentOS Linux 8" /etc/os-release` and here we do something else, be consistent :sweat_smile: